aws/client.go

```diff
@@ -0,0 +1,302 @@
+package aws
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"text/template"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsConfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
```

Comment:
```suggestion
	awstypes "github.com/aws/aws-sdk-go-v2/service/ec2/types"
```

Can you do a rename of this?

Also, maybe you could group the imports
```
goimports -w -local github.com/calyptia/ .
```